nibabel/gifti/tests/test_gifti.py

```diff
@@ -67,6 +66,17 @@ def test_dataarray():
         da = GiftiDataArray.from_array(bs_arr, 'triangle')
         assert_equal(da.datatype, data_type_codes[arr.dtype])
 
+    # Smoke test on deprecated functions
+    da = GiftiDataArray.from_array(np.ones((1,)), 'triangle')
+    with clear_and_catch_warnings() as w:
+        warnings.filterwarnings('always', category=DeprecationWarning)
+        assert_true(isinstance(da.to_xml_open(), string_types))
```

Comment:
Thanks for all your work. No worries--that one is a WIP and will take some time. I've just been looking forward to beginning that discussion!

I made the change requested. Just more sloppiness from my side. Reordered & diff looks cleaner now.